server/services/interventionsService.ts

```diff
@@ -319,11 +319,22 @@ export default class InterventionsService {
     })) as SentReferral
   }
 
-  async getSentReferrals(token: string): Promise<SentReferral[]> {
+  async getPPSentReferrals(token: string, userId: string): Promise<SentReferral[]> {
```

Comment:
I'm normally against using abbreviations in method names/business logic, as I find it impacts readability or might not make sense to newcomers to the codebase, but I'm not sure in this case - I _think_ PP/SP will make sense to everyone and we reference it in a few places e.g. comments, but you never know.

I'd probably vote in favour of making this the full `getProbationPractitionerSentReferrals` (and the same for the SP one), but am not too fussy about this, so happy to let you decide what you think is best.

Reply:
updated based on convo in dev chat